examples/geospatial/extended_description.rst

```diff
@@ -3,6 +3,16 @@
 Geospatial Examples Description
 -------------------------------
 
+NetworkX used to provide functions for reading and writing
```

Comment:
NX still provides these functions. And technically, the old versions of NX will continue to provide them. :} 
I know.... picky picky...  How about a lead in sentence like:
`Functions for reading and writing shapefiles are provided in NetworkX versions <3.0.`